packages/quill-marking-logic/src/libs/graders/sentence_combining_integration.spec.ts

```diff
@@ -163,7 +163,7 @@ describe('The checking a sentence combining question', () => {
       const matchedResponse = checkSentenceCombining(responses[0].question_uid, questionString, responses, focusPoints, incorrectSequences, responses[0].question_uid);
       assert.equal(matchedResponse.feedback, spellingFeedbackStrings[matchedResponse.author]);
       assert.equal(matchedResponse.spelling_error, true);
-      assert.equal(matchedResponse.misspelled_words[0], 'zings')
+      assert.equal(matchedResponse.misspelled_words, undefined)
```

Comment:
I'm having trouble making sense of this test compared with the new test on line [177](https://github.com/empirical-org/Empirical-Core/pull/7997/files#diff-4c8ca9f2ab436d457486284f86c65fd106024218321f110b11af265cf0cd9337R177).  If this is undefined now, how is it different from the new test.

Reply:
You're right -- it's a redundant test. I've removed the one on line 177 and kept this one in.